website/docs/r/data_export_rule.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+subcategory: "Data Export Rules"
+layout: "lacework"
+page_title: "Lacework: lacework_data_export_rule"
+description: |-
+  Create and manage Lacework Data Export Rules
+---
+
+# lacework\_data\_export\_rule
+
+Use this resource to export data collected from your Lacework account.
+For more information, see the [Data Export Rules documentation](https://docs.lacework.com/console/category/data-shares--export).
+
+## Example Usage
+
+#### Data Export Rule with S3 Data Export Channel
+```hcl
+resource "lacework_data_export_rule" "example" {
+  name             = "Data Export Rule From Terraform Updated"
+  integration_ids  = ["INT_ABC123AB385C123D4567AB8EB45BA0E7ABCD12ABF65673A"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The data export rule name.
+* `integration_ids` - (Required) The list alert channel ids for the rule to use.
```

Comment:
Can we make sure the user knows that the only Alert channel they can use is a S3 data export channel?

I would suggest to have them both as a second example